### PR TITLE
ARUHA-277: add checkstyle for acceptance tests

### PR DESCRIPTION
**Pull request by v1ctor**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -309,5 +309,5 @@ checkstyle {
 }
 
 task checkstyle {
-    dependsOn checkstyleMain, checkstyleTest
+    dependsOn checkstyleMain, checkstyleTest, checkstyleAcceptanceTest
```
Comment (v-stepanov) [inline build.gradle]: just curious, where `checkstyleAcceptanceTest` is defined?

Comment (v1ctor) [inline build.gradle]: Checkstyle gradle plugin has a task per every source set:
https://docs.gradle.org/current/userguide/checkstyle_plugin.html

**Comment by adyach** [comment]
👍

File: build.gradle (hunk 1 above)
Comment (v-stepanov) [inline build.gradle]: ah ok. That's cool.

**Comment by v-stepanov** [comment]
:shipit:

**Pull request merged**